packages/fether-react/src/Accounts/CreateAccount/AccountImportOptions/AccountImportOptions.js

```diff
@@ -67,6 +90,22 @@ class AccountImportOptions extends Component {
     }
   };
 
+  hasExistingAddressForImport = addressForImport => {
+    const { accounts } = this.props;
+    const isExistingAddress = accounts
+      .map(address => address && address.toLowerCase())
+      .includes(addressForImport);
```

Comment:
I would put `.toLowecase()` on this line. So that the input argument can be in any case.